app/services/idv/aamva_state_maintenance_window.rb

```diff
@@ -0,0 +1,165 @@
+# frozen_string_literal: true
+
+module Idv
+  class AamvaStateMaintenanceWindow
+    MAINTENANCE_WINDOWS = {
+      'CA' => [
+        # Daily, 4:00 - 5:30 am. ET.
+        { cron: '0 4 * * *', duration: 90 },
+        # Monday, 1:00 - 1:45 am. ET
+        { cron: '0 1 * * Mon', duration: 45 },
+        # Monday, 1:00 - 4:30 am. ET on 1st and 3rd Monday of month.
+        { cron: '0 1 * * Mon#1', duration: 3.5 * 60 },
+        { cron: '0 1 * * Mon#3', duration: 3.5 * 60 },
```

Comment:
These are not perfectly optimized: the every-Monday rule on line 10 is superseded by these, for example, so you could argue that line 10 should only be the 2nd, 4th, and 5th (if present) Monday. But because we just look at whether any window contains the current time, there's not really a good reason to make it that complicated.